src/scripts/components/wrapped-element.js

```diff
@@ -35,14 +35,13 @@ export default class WrappedElement {
       this.element.setAttribute('data-choice-orig-style', origStyle);
     }
 
-    this.element.setAttribute('aria-hidden', 'true');
     this.element.setAttribute('data-choice', 'active');
   }
 
   reveal() {
     // Reinstate passed element
     this.element.classList.remove(this.classNames.input);
-    this.element.classList.remove(this.classNames.hiddenState);
+    this.element.hidden = false;
```

Comment:
It's generally bad practice to mix simplified accessors (like `.hidden` or '.tabIndex`) with `getAttribute/setAttribute`. It does the same job, but confuses developers. For example in this case `this.element.hidden = false` will actually *remove* the attribute from DOM element, because `hidden` is a boolean attribute.

PS: You may push something into this PR, so, GitHub Action will re-run again with Codecov coverage report (it doesn't give access to secrets for forked repositories, as mine)

Reply:
Ah didn't realise that will remove it - fair enough 👍